.github/ISSUE_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@
 ✖ USAGE QUESTIONS → Use these dedicated resources:
       Docs - http://react.semantic-ui.com
       Chat - https://gitter.im/Semantic-Org/Semantic-UI-React
-      SO - http://stackoverflow.com/search?q=semantic-ui-react
+      SO - https://stackoverflow.com/questions/tagged/semantic-ui-react?sort=votes
```

Comment:
We have our own tag on SO, so I think it will be better if we will point users directly there